tests/testthat/test-filter_exist.R

```diff
@@ -0,0 +1,65 @@
+# filter_exist ----
+## Test 1: filter_exist() works as expected ----
+test_that("filter_exist Test 1: filter_exist() works as expected", {
+  input_dataset <- tribble(
```

Comment:
```suggestion
  input_dataset <- tibble::tribble(
```

This is one of those R package quirks.  Since we don't `library(dplyr)` in the test script and we don't `importFrom` tibble we need to directly call it from the package with `tibble::tribble`

Reply:
Ah okay, thanks for pointing that out @bms63!